test/fuse/linux/fuse_base.cc

```diff
@@ -161,15 +161,12 @@ int FuseTest::ReceiveExpected() {
   }
   offset += header_len;
 
-  // if error != 0 in fuse_out_header, this response will not have payload
-  if (((struct fuse_out_header*)buf_)->error == 0) {
-    if (read(set_expected_[0], (char*)buf_ + offset, payload_len) !=
-        ssize_t(payload_len)) {
-      std::cerr << "read fuse_out payload failed" << std::endl;
-      success = 0;
-    }
-    offset += payload_len;
+  if (read(set_expected_[0], (char*)buf_ + offset, payload_len) !=
```

Comment:
Not sure if this commit is a good place to change this?

Reply:
changed to new test framework